debian/cloud-init.postinst

```diff
@@ -327,6 +327,22 @@ fix_lp1889555() {
     db_set grub-pc/install_devices_empty "false"
 }
 
+change_cloud_init_output_log_permissions() {
+    # As a consequence of LP: #1918303
+    local oldver="$1" last_bad_ver="21.1-0ubuntu1"
+    dpkg --compare-versions "$oldver" le-nl "$last_bad_ver" || return 0
+
+    output_file="/var/log/cloud-init-output.log"
+    if [ -f "$output_file" ]; then
+        if grep -q adm /etc/group; then
```

Comment:
On building this, I'm seeing this lintian warning: https://lintian.debian.org/tags/maintainer-script-should-not-parse-etc-passwd-or-group.html

We should modify this to use `getent` per the recommendation there.